src/ping-thread.coffee

```diff
@@ -0,0 +1,7 @@
+module.exports = (robot) ->
+	robot.hear /@(thread|t)\b/i, (context) ->
+		if context.robot.adapter.fetchThreadParticipants?
+			context.robot.adapter.fetchThreadParticipants context, (list) ->
+				list = list.filter (item) -> item not in ['@thread', '@t']
```

Comment:
Alex feels strongly for `@t` so I'd keep it, although I'll probably stick to `@thread` - if nothing else I've got a feeling you'll have to really fight auto-complete to leave `@t` as just `@t`